content/docs/learning-guide/tutorials/physx/wind-provider.md

```diff
@@ -0,0 +1,103 @@
+
+    {{< image-width src="/images/learning-guide/tutorials/physx/wind-force-region-setup.png" width="900" alt="Setting up a PhysX collider for a force region." >}}
+
+## Add a Cloth prefab
+
+1. To test the wind provider, add prefab with an NVIDIA Cloth mesh. In **Asset Browser**, navigate to `Gems\NvCloth\Assets\prefabs\Cloth`, locate the `cloth_locked_edge.prefab`, and then **drag** it into the viewport.
+
+1. Use the **Move** tool to place the cloth prefab. If you are using the **Local wind tag** property, you must place the cloth asset inside the PhysX Collider volume of the wind provider entity.
+
+    ![Positioning the cloth prefab in the wind provider entity.](/images/learning-guide/tutorials/physx/wind-provider-cloth-prefab.png)
+
+1. With the prefab in position, you can hide the wind provider entity. In Entity Outliner, click the **Show/Hide Entity** toggle in the column to the right of the wind provider entity to hide it.
+
+1. The `cloth_locked_edge.prefab` has the local wind property enabled on its **Cloth** component. This generates a local wind force from the Cloth component that overrides the wind force from the wind provider entity you created. The following steps disable the local wind force of the prefab so that you can view the results of the wind provider you created:
+        
+    1. In Entity Outliner, double click the {{< icon "prefab.svg" >}} **cloth_locked_edge** prefab to edit it in Focus Mode.
+
+    2. Click the {{< icon "entity.svg" >}} **cloth_locked_edge** child entity to select it.
+
+        ![Open the cloth prefab for editing in Focus Mode.](/images/learning-guide/tutorials/physx/edit-cloth-locked-edge-prefab.png)
+
+    3. With the {{< icon "entity.svg" >}} **cloth_locked_edge** entity selected, in Entity Inspector, in the Cloth component, expand the **Wind** property group and disable the **Enable local wind velocity** toggle.
+
+        {{< image-width src="/images/learning-guide/tutorials/physx/disable-local-wind-velocity.png" width="450" alt="Setting up a PhysX collider for a force region" >}}
+
+## Test the wind simulation
+
+Finally, you can test the simulation. With the **cloth_locked_edge** prefab open for editing in Focus Mode, do the following:
+
+1. At the top of the Cloth component, enable the **Simulate in editor** toggle. As the simulation begins, the cloth object might flip and stretch wildly, but it quickly settles into a breezy wind simulation.
+
+1. With the toggle enabled, the wind simulation plays while in editor mode and you can adjust the simulation by modifying the various properties of the Cloth component. Try adjusting the following properties in the Cloth component:
+
+    * **Air drag coefficeint**
+    * **Air lift coefficeint**
+    * **Air Density**
+
+    {{< image-width src="/images/learning-guide/tutorials/physx/cloth-simulate-in-editor.png" width="450" alt="Simulating Cloth in editor." >}}
+
+The following video shows the results of the above Cloth component settings.
```

Comment:
```suggestion
The following video shows the results of using the Cloth component settings shown in the preceding image.
```
Edits for accessibility considerations:
* Expanded the description slightly to clarify that the settings referenced appear in the image (not written out).
* Dropped "above", since a reader using a screen reader might not make the same spatial distinction.